### PR TITLE
Added /enum endpoint (Apidoc 519)

### PR DESCRIPTION
**Pull request by EDDYMENS**
# Description

# New 

* Added `/enum` endpoint to provide an up-to-date list of all available options for enum parameters 

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/10dlc.yml [hunk 1]
```diff
@@ -545,6 +545,26 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/VettingProviders"
+  /enum:
```
Comment (pardel) [inline definitions/10dlc.yml:548]: Can we rename this to '/values'?

File: definitions/10dlc.yml [hunk 2]
```diff
@@ -545,6 +545,26 @@ paths:
...
+      operationId: getEnumValues
+      responses:
+        '200':
+          description: List of all enumeration values
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
```
Comment (pardel) [inline definitions/10dlc.yml:563]: Can we add an example here?

File: definitions/10dlc.yml [hunk 3]
```diff
@@ -2957,6 +2977,8 @@ tags:
     description: APIs relating to working with Campaigns
   - name: Numbers
     description: APIs relating to working with Numbers in Campaigns
+  - name: Enums
+    description: The list of options available for enum properties across all APIs
```
Comment (pardel) [inline definitions/10dlc.yml:2981]: "options available" => "values possible"

**Review by pardel — CHANGES_REQUESTED** [review]

File: definitions/10dlc.yml (hunk 1 above)
Comment (pardel) [reply definitions/10dlc.yml:548]: 👍

File: definitions/10dlc.yml (hunk 2 above)
Comment (EDDYMENS) [reply definitions/10dlc.yml]: Example added

File: definitions/10dlc.yml (hunk 3 above)
Comment (EDDYMENS) [reply definitions/10dlc.yml:2991]: Rewording applied

File: definitions/10dlc.yml [hunk 4]
```diff
@@ -1416,6 +1430,22 @@ components:
...
+      description: List of all enumeration values
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              entity_type:
+                type: array
+                items:
+                 type: object
```
Comment (pardel) [inline definitions/10dlc.yml:1443]: Are you sure this is right? I would expect an array of values not of objects.

File: definitions/10dlc.yml (hunk 3 above)
Comment (pardel) [reply definitions/10dlc.yml:2991]: I'm not seeing the change...

**Review by pardel — CHANGES_REQUESTED** [review]

File: definitions/10dlc.yml (hunk 4 above)
Comment (EDDYMENS) [reply definitions/10dlc.yml:1443]: @pardel yep it's correct, as per the output I got from my test.
Screenshot 2022-11-23 at 15 11 06

File: definitions/10dlc.yml (hunk 3 above)
Comment (EDDYMENS) [reply definitions/10dlc.yml]: Had it in two places. It should show up now.

**Review by pardel — APPROVED** [review]

**Pull request merged**